benches/benches/bevy_math/bezier.rs

```diff
@@ -0,0 +1,95 @@
+use criterion::{black_box, criterion_group, criterion_main, Criterion};
```

Comment:
Thanks for adding proper benchmarks for this! Really helpful to have better perf checks for our more complex math operations (outside of what glam already provides).

Reply:
I spent a while attempting to optimize the Bezier functions, so I needed some proper benchmarks. 😄